yoti-sdk-api/src/test/java/com/yoti/api/client/docs/session/create/filters/OrthogonalRestrictionsFilterTest.java

```diff
@@ -79,4 +79,13 @@ public void shouldBuildBlacklistedDocumentTypeRestriction() {
         assertThat(result.getTypeRestriction().getDocumentTypes(), containsInAnyOrder("PASSPORT", "DRIVING_LICENCE"));
     }
 
+    @Test
```

Comment:
Same here please (we want to ensure that the SDK is not providing default values)